examples/gravitational_wave_rates.yml

```
@@ -1,7 +1,7 @@
 cosmology: !astropy.cosmology.default_cosmology.get
 z_range: !numpy.arange [0, 2.01, 0.1]
 mag_lim: 30
-fsky: 0.0000242407
+sky_area: !astropy.units.Quantity ['1 deg2']
```

```suggestion
sky_area: 1 deg2
```
PR #316 has been merged so if you update your branch this syntax should now work. Also in the other example config files.